tests/lean/interactive/userWidget.lean

```diff
@@ -0,0 +1,15 @@
+import Lean
+open Lean Widget
+
+@[widget]
+def widget1 : UserWidgetDefinition := {
+  name := "my fancy widget"
+  javascript:= "
+  import * as React from 'react';
+  export default function (props) {
+    return React.createElement('p', {}, 'hello')
+  }"
+}
+
+#widget widget1 (Json.mkObj [])
+  --^ widgets
```

Comment:
Is the reason that we can't use the actual call name here that the widget API consists of RPC calls, which we don't generally support in the test script?

Reply:
the problem was that you have to perform two calls to rpc, and the input to the second call depends on hash so it's easier to write with it's own run handler